### ISSUE TITLE
Quake mode does not consider vertical taskbar width

### ISSUE THREAD

**Issue by Lyrex** [microsoft/terminal#10999]
### Windows Terminal version (or Windows build number)

10.0.19042.1165, 1.9.1942.0

### Steps to reproduce

1. Make sure you have at least one monitor with a vertical task bar.
2. Open a Quake mode window on the monitor with the vertical task bar.

### Expected Behavior

The Quake mode window should not be partially hidden behind the vertical task bar. It should recognize a vertical task bar is used and resize and reposition accordingly.

### Actual Behavior

One side of the terminal looks cut off by the sidebar and it seeems to be positioned behind the task bar.

![screenshot of the current behavior](https://user-images.githubusercontent.com/2445674/130317057-fd59abcd-4be4-4459-bf53-65dff877103a.png)

**Comment by zadjii-msft** [comment]
Good catch! I guess I assumed that the rectangle I was using was the `work` rect, not the `monitor` one.
